### PR TITLE
To be more explicit regarding transpilation

### LINKED ISSUE #2311: Module resolution for gulpfile.ts directory
The [docs](https://github.com/gulpjs/gulp/blob/master/docs/getting-started/2-javascript-and-gulpfiles.md) describe module resolution to a folder named gulpfile.js.  Somewhere there should be a mention that this will not work with a folder gulpfile.ts when using gulp-typescript even if it is a given that node module resolution works the same here.

### PR DESCRIPTION
**Pull request by tonyhallett**
closes #2311

### CODE REVIEW

**Comment by phated** [comment]
@tonyhallett do you have to name both things?  I would guess the actual files would be the only thing that needs the extension change.

**Comment by tonyhallett** [comment]
@phated Ok.  Can I just create another commit on the patch branch ?

**Comment by phated** [comment]
Yes, that is fine.

**Comment by tonyhallett** [comment]
Ok will do

File: docs/getting-started/2-javascript-and-gulpfiles.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ Many users start by adding all logic to a gulpfile. If it ever grows too big, it
 
 Each task can be split into its own file, then imported into your gulpfile for composition. Not only does this keep things organized, but it allows you to test each task independently or vary composition based on conditions.
 
-Node's module resolution allows you to replace your `gulpfile.js` file with a directory named `gulpfile.js` that contains an `index.js` file which is treated as a `gulpfile.js`. This directory could then contain your individual modules for tasks.
+Node's module resolution allows you to replace your `gulpfile.js` file with a directory named `gulpfile.js` that contains an `index.js` file which is treated as a `gulpfile.js`. This directory could then contain your individual modules for tasks.  If you are using a transpiler then name the folder and file accordingly.
```
Comment (janiceilene) [inline docs/getting-started/2-javascript-and-gulpfiles.md]:
```suggestion
Node's module resolution allows you to replace your `gulpfile.js` file with a directory named `gulpfile.js` that contains an `index.js` file which is treated as a `gulpfile.js`. This directory could then contain your individual modules for tasks.  If you are using a transpiler, name the folder and file accordingly.
```

Tiny change!

Comment (phated) [reply docs/getting-started/2-javascript-and-gulpfiles.md]: Thanks @janiceilene! @tonyhallett it doesn't seem like you allowed gulp contributors to make changes so you'll need to accept this suggestion then I can merge.

**Comment by phated** [comment]
Thanks @tonyhallett!

**Pull request merged**